src/components/FullDetailView/InnerDataSection.js

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation and others. Licensed under the MIT license.
+// SPDX-License-Identifier: MIT
+import React from 'react'
+import PlaceholderRenderer from '../Renderers/PlaceholderRenderer'
+import MonacoEditorWrapper from '../MonacoEditorWrapper'
+
+const InnerDataSection = ({ value, name, type = 'json', actionButton = null }) => {
```

Comment:
Why did you shift to the non-class syntax for these components (here and elsewhere). The rest of the code uses classes. Consistency wins!